app/Platform/Controllers/GameDevInterestController.php

```diff
@@ -20,9 +20,6 @@ public function __invoke(Request $request): View
         }
 
         $permissions = $request->user()->getAttribute('Permissions');
-        if ($permissions < Permissions::Developer) {
```

Comment:
This should still restrict access to JuniorDevelopers or non-developers could access the page directly via a URL.

Reply:
Good catch, resolved in https://github.com/RetroAchievements/RAWeb/pull/1896/commits/bd7b2d4bc260bca71729ba8d7c4dd2967669b613.